resources/views/front-page.blade.php

```diff
@@ -2,7 +2,10 @@
 
 @section('content')
 	<span itemscope itemtype="http://schema.org/ItemPage">
-	  <div class="row">
+	  <div class="container-fluid">
```

Comment:
would prefer to see a banner outside of the content area — would it make more sense to add it to `header.blade.php`